src/smt/command.cpp

```diff
@@ -1427,6 +1440,9 @@ void DefineFunctionRecCommand::invoke(SmtEngine* smtEngine)
 Command* DefineFunctionRecCommand::exportTo(
     ExprManager* exprManager, ExprManagerMapCollection& variableMap)
 {
+  Unimplemented();
+
+#if 0
```

Comment:
Why not remove this? Do you plan to reenable it?

Reply:
Yes, so we will have to decide whether exporting commands is a useful feature and if yes, we will have to implement a feature to export terms from one `Solver` instance to another. 

Another way to temporarily solve this problem would be to add a second overload of `exportTo` that takes a `Solver` instance and an `ExprManagerMapCollection`. The default implementation would simply get the `ExprManager` and call the current variant. For the commands that use the new solver interface, we could then export the `Expr`s of the `Term`s of `d_solver`, wrap them in `Term` instances belonging to the solver that we export to. What do you think?

Reply:
If we don't need it, we can remove it. We can always add this later if we actually need it.